kouncil-backend/src/main/java/com/consdata/kouncil/model/admin/Function.java

```diff
@@ -0,0 +1,43 @@
+package com.consdata.kouncil.model.admin;
+
+import java.util.Set;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Table(name = "FUNCTION")
+@Getter
+@Setter
+public class Function {
```

Comment:
IMO Function is very common name, maybe it better to name Feature or Functionality?

Reply:
Called this entity `SystemFunction`